src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -27,17 +29,21 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 500 Internal Server Error", 500, "Internal Server Error")]
         [InlineData("HTTP/1.1 555 we just don't like you", 555, "we just don't like you")]
         [InlineData("HTTP/1.1 600 still valid", 600, "still valid")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
```

Comment:
Per the removed comment and per top post in #24713 the following 6 tests were passing on .NET Core (Windows) at certain point. Was that incorrect statement? Did you check behavior on .NET Core 2.0? (if we didn't regress them on .NET Core since then)

Reply:
Bases on comment in the code, those test data were passing before.

But now those test data are failing (regression/improvement). I will check the behavior on .NET Core 2.0.

Reply:
I checked the behavior on .NET Core 2.0, these test cases will incorrectly pass on 2.0. This is not a regression because test data are not correct.

Reply:
Can you clarify? What does it mean that the test data is not correct?

Reply:
Please see my comment here: https://github.com/dotnet/corefx/pull/26525#issuecomment-359622781
Case 1 through 5.

Reply:
You're saying the test data is wrong because they don't conform to the RFC? That may be true, but if they used to pass and now they don't, it's still a regression (as is most any bug fix); it just may be one we decide to accept.

Reply:
Case 1 and 2: test data is wrong (should contain spaces in status description field). The expected values are wrong. So these two failing test cases indicate we made a bugfix.
Case 3 through 5 are hard to say. I don't know what should be the expected behavior, but now the current behavior (throw HttpRequestException) is consistent across all platforms.